notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>About the plots below:</p><ul><li>Since we repeat the plotting steps three times, it might de-clutter things if we turn it into a function for ourselves</li><li>After each plot, there should be a little commentary. Do we see expected spectral features? Does this star look typical for its class? Is the data noisy? You don't need to talk about all of those things, but a small, relevant comment would be good</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>